target/linux/rockchip/image/armv8.mk

```diff
@@ -47,25 +43,37 @@ define Device/friendlyarm_nanopi-r4s
   DEVICE_MODEL := NanoPi R4S
   DEVICE_VARIANT := 4GB LPDDR4
   SOC := rk3399
-  IMAGE/sysupgrade.img.gz := boot-common | boot-script | pine64-img | gzip | append-metadata
   DEVICE_PACKAGES := kmod-r8169
 endef
 TARGET_DEVICES += friendlyarm_nanopi-r4s
 
+define Device/friendlyarm_nanopi-r5c
+  DEVICE_VENDOR := FriendlyARM
+  DEVICE_MODEL := NanoPi R5C
+  SOC := rk3568
+  DEVICE_PACKAGES := kmod-r8169 kmod-rtw88 rtl8822ce-firmware wpad-basic-wolfssl
```

Comment:
in the REP, it is now prefer using `wpad-basic-mbedtls` instead of wpad-basic-wolfssl

Reply:
Updated, thanks.